aidants_connect_web/tests/test_views.py

```diff
@@ -32,6 +32,28 @@ def test_root_url_triggers_the_homepage_template(self):
         self.assertTemplateUsed(response, "aidants_connect_web/home_page.html")
 
 
+class LogoutPageTests(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        self.user = User.objects.create_user(
+            "Thierry", "thierry@thierry.com", "motdepassedethierry"
+        )
+
+    def test_logout_url_triggers_the_logout_view(self):
+        found = resolve("/logout/")
+        self.assertEqual(found.func, logout_page)
+
+    def test_logout_url_triggers_loging_if_not_logged_in(self):
+        response = self.client.get("/logout/")
+        self.assertRedirects(response, "/accounts/login/?next=/logout/")
```

Comment:
Does that mean that if you are not logged in:
- You're led to the login page
- Once you're logged in, you're being immediately logged out?

Reply:
Yes, because we want to authenticate the user and get their information.
Then, we want to be able to franceconnect through another mean (Aidants_Connect FI)

If we keep the first FranceConnect Session, then the Aidant won't be able to connect through *Aidants_Connect FI* until the first FranceConnect Session has expired.

Reply:
It means that:
- If you **are not** logged in AND you go to the logout url : you are redirected to the login page (this I the same behavior as all other "@login_required" view.
- If you **are** logged in AND you go to the logout url : you are redirected to the home page (this is the behavior described in the `settings.py` for the logout redirect"

Reply:
Oh ok then! 👍